admin-manual/installation/linux/ubuntu-xenial.rst

```diff
@@ -70,12 +76,18 @@ Ubuntu doesn't provide a package but you can download it directly from the
 `Elasticsearch site <https://www.elastic.co/downloads/elasticsearch>`_ if you
 are unable to download it using the method that follows.
 
-First, make sure that `Java <https://www.java.com/en/>`__ is installed. In this
+Make sure that `Java <https://www.java.com/en/>`__ is installed. In this
 example we are going to use OpenJDK but Oracle's JVM would also work.
 
+   code content
+
 .. code-block:: bash
 
-   sudo apt install openjdk-8-jre-headless software-properties-common
+   sudo add-apt-repository ppa:openjdk-r/ppa
+   sudo apt-get update
+
+   sudo apt install openjdk-11-jre-headless software-properties-common
```

Comment:
it looks like this got reversed. bionic is 18.04; xenial is 16.04 - so this one should still be `openjdk-8`